lib/rubocop/cop/style/safe_navigation.rb

```diff
@@ -155,6 +155,9 @@ def on_if(node)
           removal_ranges = [begin_range(node, body), end_range(node, body)]
 
           report_offense(node, method_chain, method_call, *removal_ranges) do |corrector|
+            if checked_variable.respond_to?(:safe_navigation?) && checked_variable.safe_navigation?
+              corrector.replace(receiver, checked_variable.source)
+            end
```

Comment:
Can you refactor to the following?

```suggestion
            corrector.replace(receiver, checked_variable.source) if checked_variable.csend_type?
```